tests/CellTest.java

```diff
@@ -0,0 +1,33 @@
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class CellTest {
+
+    @Test
+    public void newCellIsDeadByDefault(){
+        Cell cell = new Cell();
+
+        assertFalse(cell.getIsAlive());
+    }
+
+    @Test
+    public void whenCellIsDeadChangeStateMakesCellAlive(){
+        Cell cell = new Cell();
+        cell.changeState();
+
+        assertTrue(cell.getIsAlive());
+    }
+
+//    could be a false positive in this test, as the cell is dead in the first place
+    @Test
+    public void whenCellIsAliveChangeStateMakesCellDead(){
```

Comment:
We can think about ways to ensure a cell starts off the way we want it to be, so we can check the result of this and ensure we always know the state we're moving to. Let's do that in person later.

Reply:
sounds good